kit/Kit.cpp

```diff
@@ -1907,7 +1913,8 @@ void TraceEvent::emitOneRecordingIfEnabled(const std::string &recording)
     if (singletonDocument == nullptr)
         return;
 
-    singletonDocument->sendTextFrame("forcedtraceevent: \n" + recording);
```

Comment:
Another thing we can do is use the thread-safe "callback in a poll thread" thing - so:

whateverPoll->addCallback( { singletonDocument->sendTextFrame(); });

would presumably work fine (?) that should be sent next time that poll spins in the right thread.